dspace-spring-rest/src/main/java/org/dspace/app/rest/repository/ItemRestRepository.java

```diff
@@ -78,6 +95,20 @@ public ItemRest findOne(Context context, UUID id) {
         return page;
     }
 
+    @Override
+    public void patch(Context context, HttpServletRequest request, String apiCategory, String model, UUID uuid, Patch
+            patch)
+            throws PatchUnprocessableEntityException, PatchBadRequestException, SQLException, AuthorizeException {
+
+        // temporarily turn off authorization
+        context.turnOffAuthorisationSystem();
```

Comment:
This is probably not needed. I have discovered that the HAL browser doesn't send the Authorization header for method other than GET see https://jira.spring.io/browse/DATAREST-1077
you should try to test using postman or command line tools like curl

Reply:
You are right. The patch is successful when the Authorization header is included.  Removing the temporary fix.